File: examples/java/src/test/java/dev/selenium/elements/InteractionTest.java
package dev.selenium.elements;

import org.junit.jupiter.api.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class InteractionTest {

    @Test
    public void interactWithElements() {
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofMillis(500));
        // Navigate to Url
        driver.get("https://www.selenium.dev/selenium/web/inputs.html");

	    // Click on the element 
        WebElement checkInput=driver.findElement(By.name("checkbox_input"));
        checkInput.click();
        Boolean isChecked=checkInput.isSelected();
        assertEquals(isChecked,false);

        //SendKeys
        // Clear field to empty it from any previous data
        WebElement emailInput=driver.findElement(By.name("email_input"));
        emailInput.clear();
	    //Enter Text
        String email="admin@localhost.dev";
	    emailInput.sendKeys(email);
        //Verify
        String data=emailInput.getAttribute("value");
        assertEquals(data,email);


        //Clear Element
        // Clear field to empty it from any previous data
        emailInput.clear();
        data=emailInput.getAttribute("value");
        assertEquals(data, ""); 

        driver.quit();
    }

}

File: examples/dotnet/SeleniumDocs/Elements/InteractionTest.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;

namespace SeleniumDocs.Elements
{
    [TestClass]
    public class InteractionTest
    {
        [TestMethod]
        public void TestInteractionCommands()
        {
            IWebDriver driver = new ChromeDriver();
            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromMilliseconds(500);

            // Navigate to Url
            driver.Navigate().GoToUrl("https://www.selenium.dev/selenium/web/inputs.html");
            // Click on the element 
            IWebElement checkInput = driver.FindElement(By.Name("checkbox_input"));
            checkInput.Click();

            //Verify
            Boolean isChecked = checkInput.Selected;
            Assert.AreEqual(isChecked, false);

            //SendKeys
            // Clear field to empty it from any previous data
            IWebElement emailInput = driver.FindElement(By.Name("email_input"));
            emailInput.Clear();
            //Enter Text
            String email = "admin@localhost.dev";
            emailInput.SendKeys(email);

            //Verify
            String data = emailInput.GetAttribute("value");
            Assert.AreEqual(data, email);


            //Clear Element
            // Clear field to empty it from any previous data
            emailInput.Clear();
            data = emailInput.GetAttribute("value");
            
            //Verify
            Assert.AreEqual(data, "");

            //Quit the browser
            driver.Quit();
        }
    }
}